package.json

```diff
@@ -51,12 +51,14 @@
     "ember-cli-template-lint": "^0.7.5",
     "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-collapsible-panel": "^2.1.1",
     "ember-data": "~2.16.0",
     "ember-export-application-global": "^1.0.5",
     "ember-font-awesome": "3.0.5",
     "ember-i18n": "^5.0.2",
     "ember-load-initializers": "^0.6.0",
     "ember-page-title": "3.2.2",
+    "ember-power-select": "^2.0.0-beta.0",
```

Comment:
should add this in ember-osf dependencies

Reply:
Also, shouldn't use 2.0.0-beta.0 if not absolutely necessary (use latest 1.x instead)